applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh

```diff
@@ -4,8 +4,12 @@ cmake .. \
 -DSCOTCH_INCLUDE_DIR="/usr/include/scotch/"	            \
 -DSCOTCH_LIBRARY="libscotchmetis-5.1.so"	            \
 -DSCOTCHERR_LIBRARY="libscotcherr-5.1.so"	            \
--DCMAKE_CXX_FLAGS="-O3 -mavx2 -fPIC -fopenmp"               \
--DCMAKE_C_FLAGS="-O3 -mavx2 -fPIC -fopenmp"                 \
+-DCMAKE_CXX_FLAGS="-O3 -fPIC -fopenmp"               \
+-DCMAKE_C_FLAGS="-O3 -fPIC -fopenmp"                 \
```

Comment:
It seems that pretty much everything supports at least `-msse3`. Even non-intel/AMD processors in some cases.

https://en.wikipedia.org/wiki/SSE3